### PR TITLE
add instructor notes

### LINKED ISSUE #37: Update the instructor note

### PR DESCRIPTION
**Pull request by rogerkuou**
Fix #37.

Mainly adding notes on file dependencies.

### CODE REVIEW

File: _extras/guide.md [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
+### Episode 8
 
-[Instructors: add information about supporting learners with installation here]
+- It is not recommended to plot the `visual` band directly, due to its size (3 x 10980 x 10980). Please plot the `overview` as in the teaching material.
+- The clipped raster data: `raster_clip.tif` is required for later episodes.
 
-## Common problems
+### Episode 9
+
+- The calculated NDVI: `data/NDVI.tif` is required for later episodes.
```
Comment (fnattino) [inline _extras/guide.md:31]:
```suggestion
- The calculated NDVI: `NDVI.tif` is required for later episodes.
```

File: _extras/guide.md [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
+- The calculated classification identifier: `data/NDVI_classified.tif` is required for later episodes.
```
Comment (fnattino) [inline _extras/guide.md:32]:
```suggestion
- The calculated classification identifier: `NDVI_classified.tif` is required for later episodes.
```

File: _extras/guide.md [hunk 3]
```diff
@@ -1,33 +1,39 @@
...
+  - Do not skip this exercise;
+  - Think twice when you would like to change the query arguments in this exercise;
+  - Make sure all the audience have the output `search.json` before continuing.
 
-[Instructors: add information about timing and selecting materials as these workshops are piloted]
+### Episode 7
 
-## Technical tips and tricks
+- Loading the crop fields polygons `data/brpgewaspercelen_definitief_2020.gpkg` is not recommended, because it takes quite some time (about several minutes), and may crush the Jupyter terminal.
+- The cut version of fields: `data/cropped_field.shp` is required for later episodes.
```
Comment (fnattino) [inline _extras/guide.md:22]:
```suggestion
- The cut version of fields: `cropped_field.shp` is required for later episodes.
```

**Review by fnattino — COMMENTED** [review]
Wonderful @rogerkuou . Just fixed the paths, this will make more sense when merging #57  - go ahead and merge on my side!

**Comment by rogerkuou** [comment]
Thx! Applied the comments. Will merge this after #57

**Comment by cpranav93** [comment]
All good for me! Go ahead @rogerkuou

**Pull request merged**